apps/src/templates/watchers/Watchers.jsx

```diff
@@ -93,65 +77,14 @@ const styles = {
   }
 };
 
-const AutocompleteSelector = onClickOutside(React.createClass({
-  getInitialState() {
-    return {
-      selectedOption: this.props.options.length - 1
-    };
-  },
-
-  handleClickOutside() {
-    this.props.onClickOutside();
-  },
-
-  render() {
-    // If we ever want to highlight range of matches:
-    // http://stackoverflow.com/a/2295681
-
-    return (
-      <div
-        id="autocomplete-panel"
-        style={styles.autocompletePanel}
-      >
-        {this.props.options.map((option, index) => {
-          const isSelected = index === this.props.currentIndex;
-          const selectedStyle = {
-            backgroundColor: '#cad6fa',
-            color: 'black'
-            };
-          return (
-            <div
-              key={option}
-              onClick={(e) => this.props.onOptionClicked(option, e)}
-              onMouseOver={() => this.props.onOptionHovered(index)}
-              style={Object.assign({}, styles.autocompleteOption, isSelected ? selectedStyle : {})}
-            >
-              {option}
-            </div>
-          );
-        })}
-      </div>
-    );
-  },
-
-  propTypes: {
-    currentText: React.PropTypes.string,
-    currentIndex: React.PropTypes.number,
-    options: React.PropTypes.arrayOf(React.PropTypes.string),
-    onOptionClicked: React.PropTypes.func,
-    onOptionHovered: React.PropTypes.func,
-    onClickOutside: React.PropTypes.func
-  }
-}));
-
 /**
  * A "watchers" window for our debugger.
  */
-const DebugWatch = React.createClass({
+const Watchers = React.createClass({
```

Comment:
You can just add the `export` keyword where you define the consts instead of having a single `export` at the bottom with multiple values.